README.md

```diff
@@ -1283,30 +1283,53 @@ Optional: If you have already joined managed nodes to AD and you want to skip ru
 
 For an example playbook, see [Configuring AD integration with a pre-created keytab file](#configuring-ad-integration-without-joining-to-ad).
 
+
+If you define the `ad_integration_user` variable, the role obtains the kerberos ticket for creating keytab for the `ad_integration_user@ad_integration_realm` user with the `ad_integration_password` password.
+If you don't define the `ad_integration_user` variable, the role obtains the kerberos ticket for `mssql_ad_sql_user_name@ad_integration_realm` user with the `mssql_ad_sql_password` password.
+
 Default: `true`
 
 Type: `bool`
 
-##### mssql_ad_obtain_kerberos
+##### mssql_ad_kerberos_user
 
-Optional: If you have already obtained Kerberos ticket on managed nodes and you want to skip obtaining it, you can set this variable to `false`.
+Optional, you can use a specific user for obtaining kerberos ticket for creating the `mssql_ad_sql_user_name` user and keytab.
+By default, the role uses the following logic:
 
-By default, when mssql_ad_obtain_kerberos is set to `true`, you must set `ad_integration_user` and `ad_integration_password` variables.
+1. If you define the `ad_integration_user` variable, the role obtains the kerberos ticket for the `ad_integration_user@ad_integration_realm` user with the `ad_integration_password` password.
 
-For an example playbook, see [Configuring AD integration with a pre-created keytab file](#configuring-ad-integration-without-joining-to-ad-and-without-obtaining-kerberos-ticket).
+2. If you don't define the `ad_integration_user` variable, the role obtains the kerberos ticket for `mssql_ad_sql_user_name@ad_integration_realm` user with the `mssql_ad_sql_password` password.
 
-Default: `true`
+You must provide this variable in the format <username>@{{ ad_integration_realm }}
```

Comment:
If they provide it in that format, it would result in <username>@<realm>@<realm>. In vars you set `__mssql_ad_kinit_user` to `{{ mssql_ad_kerberos_user }}@{{ ad_integration_realm | upper }}`. Either you need to have `mssql_ad_kerberos_user` specified as just a username or you need `__mssql_ad_kinit_user` to just be set to `mssql_ad_kerberos_user` and the default value of `mssql_ad_kerberos_user` to include the realm.

Reply:
Good point, I first wrote this and then fixed the code to avoid this specific format, just forgot to remove the text.